arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx

```diff
@@ -395,6 +396,140 @@ Here we can see that the gas cost incurred by the execution of the first transac
 
 To learn more about the gas cost model, see [how to estimate gas](/devs-how-tos/how-to-estimate-gas).
 
+## Option 5: Call and modify state
+
+In this example, we'll demonstrate how to _read from_ and _write to_ a precompile contract's ArbOS state.
```

Comment:
We can maybe add a link to `/arbos/#arbos-state` in "ArbOS state" for some extra reference

Reply:
I didn't apply those tags to there 3 lines is because they point to specific line, so I use a constant url here, if we use it, the file will update to the latest version which might be different line in the future